sound/hybrid_font.h

```diff
@@ -234,11 +234,38 @@ class HybridFont : public SaberBase, public Looper {
     EnableAmplifier();
     if (!f->files_found()) return RefPtr<BufferedWavPlayer>(nullptr);
     RefPtr<BufferedWavPlayer> player = GetFreeWavPlayer();
-    if (player) {
-      player->set_volume_now(font_config.volEff / 16.0f);
-      player->PlayOnce(f);
-      current_effect_length_ = player->length();
+    if (!player) {
+      STDOUT.println("Out of WAV players! Getting more...");
+      float oldest_ = 0;
+      uint32_t kill_ = 0;
+      // check wavplayers, rule out hum and smoothswings then kill oldest one 
+      for (size_t i = 0; i < NELEM(wav_players); i++) {
+        if (wav_players[i].isPlaying() && wav_players[i].current_file_id().GetEffect() == f) {
+          float pos = wav_players[i].pos();
+          if (pos > oldest_) {
+            oldest_ = pos;
+            kill_ = i;
+          }    
+        } else {
+          // unsuccessful acquisition. Let's try again.
```

Comment:
This comment doesn't make any sense.
I don't see any "trying again" here.

Reply:
Well this was the fix for freezing and squee of death.
For some reason, it just doesn't get it sometimes and we get to here, so "try again" meaning give up this time and go back to listening for PlayPolyphonic().
Which, in testing, is an  imperceivable skip.

Reply:
removed.